plugins/OAuth2/views/settings.php

```diff
@@ -0,0 +1,25 @@
+<h1><?php echo $this->data('Title'); ?></h1>
+
+<div class="PageInfo">
+    <h2><?php echo t('Create Single Sign On integration using the OAuth2 protocol!'); ?></h2>
+
+    <p>
```

Comment:
I think this settings page is already incorrect for the new dashboard. I also question why we are nesting `p` tags here.